CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## TBD
+### Fixed 
+* Bug where not all entries were picked up due to deletions
+* Bug in folder.isEmpty call for S3AFileSystem they would incorrectly label a folder empty if it had Hadoop "<directoryname>_$folder$" objects
```

Comment:
folder.isEmpty -> `folder.isEmpty`
they -> that

Reply:
and fullstops at the end of the sentences :)

Reply:
ah yeah forgot to go over this after I just did a brain dump, thanks! :)